Package.swift

```diff
@@ -28,7 +28,7 @@ let package = Package(
             path: "HCaptcha",
             exclude: ["Classes/Rx"],
             resources: [
-                .process("Assets/hcaptcha.html")
+                .process("Resources/PrivacyInfo.xcprivacy")
```

Comment:
was this deletion intentional?

Reply:
Nope, I missed to do this before, because we moved `Assets/hcaptcha.html` into autogenerated `HCaptchaHtml.swift` we should not keep `Assets/hcaptcha.html` resource